src/pages/nfts/profile/[accountAddress]/achievements.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRouter } from 'next/router'
+import React from 'react'
+import { useAchievementsForAddress, useProfileForAddress } from 'state/profile/hooks'
+import { NftMarketLayout } from 'views/Nft/market/Layout'
+import NftProfile from 'views/Nft/market/Profile'
+import Achievements from 'views/Nft/market/Profile/components/Achievements'
+
+const NftProfileAchievementsPage = () => {
+  const accountAddress = useRouter().query.accountAddress as string
+  const { profile: profileHookState } = useProfileForAddress(accountAddress)
+  const { profile } = profileHookState || {}
+  const { achievements, isFetching: isAchievementFetching } = useAchievementsForAddress(accountAddress)
```

Comment:
Am I missing something or useAchievements for the connected account will not be used?

Reply:
I'm late for the party. so the only difference of connected vs unconnected here is with redux or not, right 🌚

Reply:
Yes, Unconnected uses hooks with ...ForAddress, Connected takes it from redux

Reply:
These two should eventually merge together. I will put connected/unconnected pattern back for now, try not to change too much in this PR.
About `useAchievementsForAddress` being used twice in `UnconnectedProfile` and `UnConnectedAchievement`
I can't think of any quick fix for this one.  Let's hope https://github.com/pancakeswap/pancake-frontend/pull/2965 will be merged soon